.circleci/config.yml

```diff
@@ -569,25 +547,33 @@ after-build-and-test: &after-build-and-test
 
 after-platform-builds: &after-platform-builds
```

Comment:
I think we can replace `after-platform-builds` (and remove it altogether) with `platform-build`.

Reply:
I prefer to keep it similar to the others. The other required elements are named **after-<foo>**. This matches those.